docs/overview/office-add-ins-community-call.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Microsoft Office Add-ins community call
+description: 'Information about the monthly Microsoft Office Add-ins community call agendas and schedule'
+ms.date: 03/08/2021
+localization_priority: Normal
+---
+
+# Microsoft Office Add-ins community call
+
+The Microsoft Office Add-ins community call is a monthly call where you can learn more about new features, development practices, and additional information about creating Office Add-ins. The next call is Wednesday, March 10, 2021 at 8:00AM PDT. You can download the calendar invite at [https://aka.ms/officeaddinscommunitycall](https://aka.ms/officeaddinscommunitycall).
```

Comment:
We might leave "next call" off this and just put a list of dates, otherwise a hassle to hand-update?

Reply:
Good point. I updated to just indicate it is second Wednesday of each month.